serverless/networking/serverless-ossm.adoc

```diff
@@ -35,7 +35,7 @@ spec:
 Adding sidecar injection to Pods in system namespaces such as `knative-serving` and `knative-serving-ingress` is not supported.
 ====
 
```

Comment:
If it's just one procedure, it should be fine to have it at the assembly level. But yeah, it should have an intro. And the prereqs should be unordered.

Reply:
Same as above @nak3 - why would a user want to use SM with Serverless? Can we just say something like:
"This section provides information about how you can configure OpenShift Serverless to use Red Hat Service Mesh as a networking layer instead of the default Kourier networking layer." ?

Reply:
Because we can use the ServiceMesh features like JWT auth, custom domain and custom TLS that we are trying to add to the docs now. These features are not achievable by serverless only.
Does it make sense?

Reply:
> If it's just one procedure, it should be fine to have it at the assembly level. But yeah, it should have an intro. And the prereqs should be unordered.

I see that another procedure module[1] was added to the `serverless-ossm.adoc` assembly. If there are two procedures in the assembly now, I think the first procedure can be its own module as well.

[1]`modules/serverless-service-mesh-enable-sidecar-injection.adoc`

Reply:
> If it's just one procedure, it should be fine to have it at the assembly level. But yeah, it should have an intro. And the prereqs should be unordered.

Following up on Bob's previous comment that the prereqs should be an unordered list.